requirements/tests

```diff
@@ -8,9 +8,9 @@ testtools; python_version < '3.10'
 
 # ASGI Specific (daphne is installed on a its own tox env)
 pytest-asyncio
-httpx; python_version >= '3.6'
-uvicorn; python_version >= '3.6'
 aiofiles; python_version >= '3.6'
+httpx; python_version >= '3.6'
+uvicorn; python_version >= '3.7'
```

Comment:
I guess you'd want to do something like this:
```python
uvicorn == 0.16.0; python_version == '3.6'
uvicorn; python_version >= '3.7'
```

Reply:
Hm yes, thanks 👍 I'll rectify this. I just push something to avoid having a broken master branch.